src/styles/getTooltipStyles.ts

```diff
@@ -95,42 +94,49 @@ function isOverlappingAtTop(xOffset, yOffset, tooltip, tooltipTargetWidth, toolt
     return isOverlappingAtTargetCenterX;
 }
 
+type TooltipStyles = {
+    animationStyle: ViewStyle;
+    rootWrapperStyle: Omit<ViewStyle, 'position'> | Pick<CSSProperties, 'position'>;
+    textStyle: TextStyle;
+    pointerWrapperStyle: Omit<ViewStyle, 'position'> | Pick<CSSProperties, 'position'>;
+    pointerStyle: ViewStyle;
```

Comment:
```suggestion
    animationStyle: ViewStyle | CSSProperties;
    rootWrapperStyle: Omit<ViewStyle, 'position'> | Pick<CSSProperties, 'position'>;
    textStyle: TextStyle | CSSProperties;
    pointerWrapperStyle: Omit<ViewStyle, 'position'> | Pick<CSSProperties, 'position'>;
    pointerStyle: ViewStyle | CSSProperties;
```

I think we should allow `CSSProperties` in other styles, shouldn't we?